shared/chat/conversation/messages/wrapper/wrapper-author/container.js

```diff
@@ -12,8 +12,8 @@ const mapStateToProps = (state: TypedState, {message, previous, innerClass, isEd
   const isYou = state.config.username === message.author
   const isFollowing = state.config.following.has(message.author)
   const isBroken = state.users.infoMap.getIn([message.author, 'broken'], false)
-  const lastReadMessageID = state.chat2.lastReadMessageMap.get(message.conversationIDKey)
-  const lastPositionExists = !!previous && lastReadMessageID === previous.id
+  const {readMsgID, maxMsgID} = Constants.getMeta(state, message.conversationIDKey)
+  const lastPositionExists = maxMsgID > readMsgID
```

Comment:
`maxMsgID > readMsgID` is the same logic as `!!previous && lastReadMessageID === previous.id` – are there messages after the last read message?

I like `maxMsgID > readMsgID` because it doesn't rely on `orangeLineMap`. I think it's better that that map be used only for displaying the orange line.